draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1901,9 +1904,66 @@ and feedback.
 All references to {{RFC4122}} in the IANA registries should be replaced with references to this document.
 References to {{RFC4122}} document's Section 4.1.2 should be updated to refer to this document's {{format}}.
 
-There is no update required to the IANA URN namespace registration {{URNNamespaces}} for UUID filed in {{RFC4122}}.
-
-IANA is not required to track UUIDs used for identifying items such as versions, variants, namespaces, or hashspaces.
+The IANA URN namespace registration {{URNNamespaces}} for UUID filed in {{RFC4122}} should be updated to reference this document.
+
+Finally IANA should track UUID Variants, UUID Subtypes, and Special Case "Reserved UUIDs" such as Min, Max, and Namespace IDs as specified in {{iana1}}, {{iana2}}, {{iana3}}.
+
+The policy for these three registries is "Specification Required", per {{RFC8126}}, Section 4.6.
+
+When evaluating requests, the designated expert(s) should consider community feedback, how well-defined is the reference specification, and this specification's requirements.
+Vendor-specific, application-specific, and deployment-specific values are unable to be registered.
+Specification documents should be published in a stable, freely available manner (ideally located with a URL) but need not be standards.
+The designated experts will either approve or deny the registration request, and communicate their decision to IANA. Denials should include an explanation and, if applicable, suggestions as to how to make the request successful.
+
+## IANA UUID Variants Registry and Registration {#iana1}
+This specification defines the "UUID Variants" registry for common, widely used UUID standards.
+
+{{ianaVariants}} should be used as-is for this registry with "This document" replaced as required.
+
+| Name           | Bits | Variant | Reference                  |
+| Apollo NCS     | 0xxx | 1-7     | {{RFC4122}}, This document |
+| OSF DCE / IETF | 10xx | 8-9,A-B | {{RFC4122}}, This document |
+| Microsoft      | 110x | C-D     | {{RFC4122}}, This document |
+| Unused         | 111x | E-F     | {{RFC4122}}, This document |
+{: #ianaVariants title='IANA UUID Variants'}
+
+## IANA UUID Subtype Registry and Registration {#iana2}
+This specification defines the "UUID Subtype" registry for common, widely used UUID standards.
+
+{{ianaSubtypes}} should be used as-is for this registry with "This document" replaced as required.
+
+| Name                           | ID | Hex | Subtype | Variant        | Reference                  |
```

Comment:
Are you talking about the column header of `Subtype` change to `Version`? 
Or the row items in the column being changed from `Version` to `version`?

I specifically used "Subtype" in the column headers since "version" is a construct of this UUID RFC and other subtyping mechanisms exist (in a historical sense at least.(

Reply:
I meant the column value `version`. The word `Subtype` as column header is good enough.